test/integration/helpers.tsx

```diff
@@ -43,6 +65,41 @@ export function mock4byte(hexSignature: string, textSignature?: string) {
   return mockEndpoint;
 }
 
+export const getSelectedAccountGroup = <
+  State extends WidenDeep<AccountTreeControllerState>,
```

Comment:
We use `WidenDeep` to avoid having to type-cast with `as unknown as AccountTreeControllerState` in the callers.

Also, it seems that loading `.json` files with `import` loads literal string values as `string` rather than there actual literal values and since the `AccountTreeControllerState` uses strict typing for group/wallet nodes (`entropy:${string}`, `keyring:${string}`, etc..), we just widen those as simple `string`.